src/ObservationDialog/Select.js

```diff
@@ -62,6 +62,19 @@ type Props = {
   options: SelectOptions
 }
 
+function Encoder (options) {
+  return {
+    toValue: (v) => {
+      var opts = options.find((ops) => ops.label === v)
```

Comment:
[`SelectOptions`](https://github.com/digidem/react-mapfilter/blob/v3/src/types.js#L210) can also be `number`, `string`, `boolean` or `null` so this (and below) requires a type check.

Reply:
OK Added tests for this and fixing now